h/views/account_signup.py

```diff
@@ -29,21 +29,21 @@ def __init__(self, request):
             css_class="js-disable-on-submit",
         )
 
-    @view_config(request_method="GET")
+    @view_config(
+        request_method="GET", renderer="h:templates/accounts/signup.html.jinja2"
+    )
     def get(self):
         """Render the empty registration form."""
         self._redirect_if_logged_in()
 
         return {"form": self.form.render()}
 
-    @view_config(request_method="POST")
+    @view_config(
+        request_method="POST",
+        renderer="h:templates/accounts/signup-successful.html.jinja2",
+    )
     def post(self):
-        """
-        Handle submission of the new user registration form.
-
-        Validates the form data, creates a new activation for the user, sends
-        the activation mail, and then redirects the user to the index.
-        """
```

Comment:
This docstring was lies, see https://github.com/hypothesis/h/pull/5964/commits/6b8999e51bae33e39d91684fa0b1852e2d390ad1

Reply:
Just for run, look at line 67 below

Reply:
Just for fun, look at line 67 below